srcalc/src/main/java/gov/va/med/srcalc/web/controller/admin/NewRuleController.java

```diff
@@ -0,0 +1,136 @@
+import gov.va.med.srcalc.service.AdminService;
+import gov.va.med.srcalc.service.DuplicateRuleNameException;
+import gov.va.med.srcalc.service.InvalidIdentifierException;
+import gov.va.med.srcalc.util.ValidationCodes;
+import gov.va.med.srcalc.web.view.VariableSummary;
+import gov.va.med.srcalc.web.view.Views;
+import gov.va.med.srcalc.web.view.admin.EditRule;
+import gov.va.med.srcalc.web.view.admin.ValueMatcherBuilder;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.servlet.ModelAndView;
+
+import com.google.common.primitives.Ints;
+
+/**
+ * <p>Web MVC controller for creating a new rule.</p>
+ */
+@Controller
+public class NewRuleController
+{
+    public static final String NEW_RULE_URL = "/admin/newRule";
+    
+    private static final Logger fLogger = LoggerFactory.getLogger(NewDiscreteNumericalVarController.class);
+    /**
+     * The model attribute key of the {@link Rule} object.
+     */
+    protected static final String ATTRIBUTE_RULE = "rule";
+    
+    private final AdminService fAdminService;
+    
+    /**
+     * Constructs an instance.
+     * @param adminService the AdminService to use for operations
+     * @throws NullPointerException if adminService is null
+     */
+    @Inject
+    public NewRuleController(final AdminService adminService)
+    {
+        fAdminService = Objects.requireNonNull(adminService);
+    }
+    
+    @ModelAttribute(ATTRIBUTE_RULE)
+    private EditRule createEditRule()
+    {
+        fLogger.trace("Creating EditDiscreteNumericalVar.");
+        return new EditRule(fAdminService);
+    }
+    
+    @RequestMapping(value = NEW_RULE_URL, method = RequestMethod.GET)
+    public ModelAndView displayForm(
+            @ModelAttribute(ATTRIBUTE_RULE) final EditRule editRule) throws InvalidIdentifierException
+    {
+        final ModelAndView mav = new ModelAndView(Views.EDIT_RULE);
+        // Provide the URL to save the rule to the view.
+        mav.addObject("SAVE_URL", NEW_RULE_URL);
+        // Note: "rule" is already in the model via createEditRule().
+        mav.addObject("isNewRule", true);
+        // A map of the variable's key to its respective summary
+        final Map<String, VariableSummary> variableSummaries = new HashMap<String, VariableSummary>();
+        for(final ValueMatcherBuilder matcher: editRule.getMatchers())
+        {
+            final Variable var = fAdminService.getVariable(matcher.getVariableKey());
+            variableSummaries.put(matcher.getVariableKey(), VariableSummary.fromVariable(var));
+        }
+        mav.addObject("variableSummaries", variableSummaries);
```

Comment:
At first I didn't understand why you added the variableSummaries to the model here, but now I get it. An explanatory comment would be helpful, but I have a design idea I'd like to discuss with you.